source/distributed/full_distributed_tria.cc

```diff
@@ -0,0 +1,508 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2008 - 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+#include <deal.II/base/memory_consumption.h>
+
+#include <deal.II/distributed/full_distributed_tria.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+// Forward declarations
+namespace GridGenerator
+{
+  template <int dim, int spacedim>
+  void
+  hyper_cube(Triangulation<dim, spacedim> &tria,
+             const double                  left,
+             const double                  right,
+             const bool                    colorize);
+} // namespace GridGenerator
+
+namespace parallel
+{
+  namespace fullydistributed
```

Comment:
the namespace and filename do not match.

Reply:
@bangerth wished to change the file name to `full_distributed_tria.cc`. It used to match the namespace (`fullydistributed_tria.cc`).

Reply:
Sorry for the back and forth. :-) I think he meant fully_distributed_tria.cc.